app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,5 @@
   <string name="cannot_open_file">Failed to open file\nPlease try looking for this file in the Device Tab of your Library</string>
   <string name="send_files_title">Send Files</string>
   <string name="receive_files_title">Receive Files</string>
+  <string name="error_occurred" translatable="false">System unable to grant permission!</string>
```

Comment:
We are adding this in core module also. We can get rid of it here. The `core`module will share the resource between `app`  and `custom` module

Reply:
move to core